README.adoc

```diff
@@ -165,7 +165,8 @@ $ make test-unit
 
 integration-tests::
 Integration tests demand more setup (i.e. the PostgreSQL DB must be already
-running) and probably time.
+running) and probably time. Please make sure you have installed psql and
```

Comment:
`psql` might be the package name for your distribution. Better write PostgreSQL.